modules/azure/api_management_api_schema/variables.tf

```diff
@@ -0,0 +1,30 @@
+variable "resource_group_name" {
+  type        = string
+  description = "Name of the resource group."
+}
+
+variable "api_management_name" {
+  type        = string
+  description = "The name of the API management service."
+}
+
+variable "schema" {
```

Comment:
since it allows to create multiple schemas, should we also allow lists of schema? rather than only 1 schema?

Reply:
or do you think thinks it best for the User just to create multiple instances of  api_schema modules?